src/dc_strencode.rs

```diff
@@ -23,7 +23,8 @@ pub fn dc_encode_header_words(input: impl AsRef<str>) -> String {
 }
 
 fn must_encode(byte: u8) -> bool {
-    static SPECIALS: &[u8] = b",:!\"#$@[\\]^`{|}~=?_";
+    // XXX do we need to put ":" in here?
+    static SPECIALS: &[u8] = b",!\"#$@[\\]^`{|}~=?_";
```

Comment:
core-c encoded [chars >=128 _plus_ backquote _plus_ `,:!"#$@[\]^{|}~=?_`](https://github.com/deltachat/deltachat-core/blob/master/src/dc_strencode.c#L133) - so yes, this includes the colon.
iirc the code came from an example shipped together with libetpan.

Reply:
we are now using a crate that does all quopri encoding  -- let's see if there are remaining RL problems.